app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1533,6 +1533,16 @@ protected void onResume() {
     }
 
 
+    public String getDefaultLocale(){
+
+        String language = Locale.getDefault().getLanguage();
```

Comment:
We don't need a separate method for this imo
Simply use `Locale.getDefault().getLanguage()` in the api call.